usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md

```diff
@@ -36,4 +36,5 @@ This endpoint returns a high-level overview of a specific state or territory, gi
     The year the median household income is based on.
 + `total_prime_amount`: 300200000000 (required, number)
 + `total_prime_awards`: 327721 (required, number)
++ `face_value_loan_guarantee`: 88888.0 (required, number)
```

Comment:
I see examples of  `face_value_loan_guarantee` in the API docs, but this feature request mirrors the new functionality in Recipient Profile (`/api/v2/recipient/duns/{recipient_id}/{?year}`) which use "total_face_value_loan_amount" and "total_face_value_loan_transactions" which is what I recommend for this endpoint

Reply:
Clarified this with @ross-williford. We're going with `total_face_value_loan_amount`